dist/Rokt-Kit.common.js

```diff
@@ -120,9 +122,18 @@ var constructor = function () {
 
         self.userAttributes = filteredAttributes;
 
-        var selectPlacementsAttributes = mergeObjects(filteredAttributes, {
-            mpid: mpid,
-        });
+        var optimizelyAttributes =
```

Comment:
Please remove the dist files from the commit. These are built by the CI/CD process.